src/app/CommandHandler.cpp

```diff
@@ -244,6 +244,20 @@ CHIP_ERROR CommandHandler::SendCommandResponse()
     return CHIP_NO_ERROR;
 }
 
+namespace {
+// We use this when the sender did not actually provide a CommandFields struct,
+// to avoid downstream consumers having to worry about cases when there is or is
+// not a struct available.
+//
+// The encoding here is:
+// Control byte: Struct, anonymous tag (since we can't use a context tag at top
+//               level, and consumers should not care about the tag here).
+// End of container.
+constexpr uint8_t sNoFields[] = {
+    0b00010101 /* struct, anonymous tag */, 0b00011000 /* End of container */
```

Comment:
Please use the TLV macros instead please. Those avoid the direct coupling to internal TLV encoding specifics:

```
constexpr uint8_t sNoFields[] = {
            CHIP_TLV_STRUCTURE(CHIP_TLV_TAG_ANONYMOUS),
            CHIP_TLV_END_OF_CONTAINER,
}
```

Reply:
Did not know about those.  Yes, that makes a lot of sense!